gui/layers/_visual_wrapper.py

```diff
@@ -32,6 +33,8 @@ class VisualWrapper:
     """
     def __init__(self, central_node):
         self._node = central_node
+        self._blending_modes = list(get_state_presets().keys())
```

Comment:
Since GitHub somehow threw out this comment (grrrrr) I'll write it up again.

This should be a class variable. Also if you make it a `set` instead you can use it in f-strings :P

Reply:
Not sure I don't understand, do you mean something like
```python
 @property
def _blending_modes(self):
    """Doc string
    """
    return list(get_state_presets().keys())
```
or with a `setter`?

Reply:
```python
class Layer:
...
_blending_modes = set(get_state_presets().keys())
```